tests/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@
 
 cmake_minimum_required(VERSION 3.0)
 enable_language(CXX)
-set(CMAKE_CXX_STANDARD 98)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
I kept an older version here as a way to indicate that the code is not using any advanced feature in higher C++ versions.

Reply:
Let me do a local test. The need for a higher C++ problem has something to do with `gtest`, but let me double check it.